chimerapy/networking/client.py

```diff
@@ -40,14 +40,14 @@ def __init__(
         id: str,
         host: str,
         port: int,
-        ws_handlers: Dict[enum.Enum, Callable[[], Coroutine]] = {},
+        ws_handlers: Dict[enum.Enum, Callable] = {},
```

Comment:
This should be?
```suggestion
        ws_handlers: Dict[int, Callable] = {},
```

Reply:
Yes, thanks for the great catch!

Reply:
`Enum` is correct here since later we convert the enum to an integer later. But `Enum` is preferable in the constructor to provide a warning for an `int` input (as handler signals are strict and limited).